app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/adapter/viewholder/viewholder.kt

```diff
@@ -2,17 +2,17 @@ package com.puzzlebench.clean_marvel_kotlin.presentation.adapter.viewholder
 
 import android.support.v7.widget.RecyclerView
 import android.view.View
-import com.puzzlebench.cmk.domain.model.Character
 import com.puzzlebench.clean_marvel_kotlin.presentation.extension.getImageByUrl
 import com.puzzlebench.clean_marvel_kotlin.presentation.listener.CharacterListener
-import kotlinx.android.synthetic.main.character_cards_layout.view.*
-
+import com.puzzlebench.cmk.domain.model.Character
+import kotlinx.android.synthetic.main.character_cards_layout.view.image_thumbnail
+import kotlinx.android.synthetic.main.character_cards_layout.view.tv_item
 
 class CharactersAdapterViewHolder(view: View, val listener: CharacterListener) : RecyclerView.ViewHolder(view) {
 
     fun bind(item: Character) = with(itemView) {
         tv_item.text = item.name
-        val string = item.thumbnail.path + "." + item.thumbnail.extension
+        val string = item.thumbnail?.path + "." + item.thumbnail?.extension
```

Comment:
please use the kotlin style string concat

"${item.thumbnail?.path}.${item.thumbnail?.extension}"